### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jlmendgom5TRAN**
Links: https://fivetran.height.app/T-680085

Fixes a 404 identified here:

404ide

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@
 # Fivetran Platform dbt Package ([Docs](https://fivetran.github.io/dbt_fivetran_log/))
 # 📣 What does this dbt package do?
 - Generates a comprehensive data dictionary of your Fivetran Platform connector (previously called Fivetran Log) data via the [dbt docs site](https://fivetran.github.io/dbt_fivetran_log/)
-- Produces staging models in the format described by [this ERD](https://fivetran.com/docs/logs/fivetran-log#schemainformation) which clean, test, and prepare your Fivetran data from [Fivetran's free connector](https://fivetran.com/docs/applications/fivetran-log) and generates analysis ready end models.
+- Produces staging models in the format described by [this ERD](https://fivetran.com/docs/logs/fivetran-platform#schemainformation) which clean, test, and prepare your Fivetran data from our free [Fivetran Platform connector](/docs/logs/fivetran-platform) and generates analysis ready end models.
```
Comment (fivetran-dejantucakov) [inline README.md:19]:
```suggestion
- Produces staging models in the format described by [this ERD](https://fivetran.com/docs/logs/fivetran-platform#schemainformation) which clean, test, and prepare your Fivetran data from our free [Fivetran Platform connector](https://fivetran.com/docs/logs/fivetran-platform) and generates analysis ready end models.
```
DBT docs work differently, so we need full URL paths

File: README.md [hunk 2]
```diff
@@ -16,7 +16,7 @@
...
 - The above mentioned models enable you to better understand how you are spending money in Fivetran according to our [consumption-based pricing model](https://fivetran.com/docs/getting-started/consumption-based-pricing) as well as providing details about the performance and status of your Fivetran connectors. This is achieved by:
```
Comment (fivetran-dejantucakov) [inline README.md:20]:
```suggestion
- The above mentioned models enable you to better understand how you are spending money in Fivetran according to our [consumption-based pricing model](https://fivetran.com/docs/usage-based-pricing) as well as providing details about the performance and status of your Fivetran connectors. This is achieved by:
```
One less redirect.

**Review by fivetran-dejantucakov — CHANGES_REQUESTED** [review]
@jlmendgom5TRAN Please see comments and suggestions

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM

**Pull request merged**